app/tests/test_database.py

```diff
@@ -304,6 +304,7 @@ def test_individual(self):
 
         data["birth_date"] = self.breeding[0].birth_date
         data["litter_size"] = self.breeding[0].litter_size
+        data["litter_size6w"] = self.breeding[0].litter_size
```

Comment:
Why does it take the data from `self.breeding[0].litter_size` and not `self.breeding[0].litter_size6w`?

Reply:
It is because I set the size6w as the same as litter_size in the test database

Reply:
I changed it so its more clear.